getting-started/templates/sizing-examples/systems-sizing-example.yaml

```diff
@@ -0,0 +1,413 @@
+    ## This is used in conjunction with presenceChangedCountLimit to signal connection events (whichever happens first)
+    presenceChangedIntervalMs: 3000
+    ## @param saltmaster.events.presenceChangedCountLimit Specifies at what count should the SaltMaster publish the collected connection events
+    ## This is used in conjunction with presenceChangedIntervalMs to signal connection events (whichever happens first)
+    presenceChangedCountLimit: 25
+
+## The systems management service registers remote systems and applies configuration via jobs.
+systems:
+  resources:
+    requests:
+      memory: 1024Mi
+      cpu: 500m
+    limits:
+      memory: 1024Mi
+  autoscaling:
+    minReplicas: 2
+    maxReplicas: 10
+    targetCPUUtilizationPercentage: 60
+    targetMemoryUtilizationPercentage: 70
+
+  ## Configure job scheduling
+  jobs:
+    ##  @param systems.jobs.maxInProgressQueuedJobs The maximum number of queued jobs that can be dispatched to managed clients at any given time.
+    maxInProgressQueuedJobs: 250
+
+  ## Configure the system feeds requests
+  systemFeeds:
+    intersection:
+      ## @param systems.systemFeeds.intersection.limit Configure the systems feeds intersection request
+      limit: 1000
+    ## Configure the systems feeds conflicts request
+    conflicts:
+      systems:
+        ## @param systems.systemFeeds.conflicts.systems.limit Upper limit of the number of system Ids for which to determine feed conflicts
+        limit: 1000
+      feeds:
+        ## @param systems.systemFeeds.conflicts.feeds.limit Upper limit of the number of feeds to be used when determining feed conflicts against system feeds
+        limit: 1000
+
+## The system state service applies pre-configured states to client systems.
+systemsstate:
+  resources:
+    requests:
+      memory: 1024Mi
+      cpu: 250m
+    limits:
+      memory: 1024Mi
+  autoscaling:
+    minReplicas: 2
+    maxReplicas: 10
+    targetCPUUtilizationPercentage: 60
+    targetMemoryUtilizationPercentage: 80
+
+  ## @param systemsstate.stateLimitPerOrg Maximum number of states
+  stateLimitPerOrg: 1000
+
+  ## @param systemsstate.stateContentLimitMb Limit in megabytes of the maximum size of a state's content
+  stateContentLimitMb: 1
+
+## The SMTP service is used by the notification service to send emails.
+smtp:
+  resources:
+    limits:
+      memory: 150Mi
+    requests:
+      cpu: 100m
+      memory: 150Mi
+  autoscaling:
+    minReplicas: 2
+    maxReplicas: 10
+    targetCPUUtilizationPercentage: 60
+    targetMemoryUtilizationPercentage: 80
+
+## The tag service publishes tags
+tags:
+  resources:
+    requests:
+      cpu: 4000m
+      memory: 8Gi
+    limits:
+      memory: 8Gi
+
+  ## @param tags.replicaCount Number of pod replicas to deploy.
+  ## This service does not have auto-scaling. Set this value according to the needs of your environment.
+  replicaCount: 1
```

Comment:
I'm wondering if we should ever recommend a single replica since it's effectively not an HA configuration with just 1. I don't believe the purpose of this doc is to describe a minimal size. Is there a consideration I am missing as to when its best practice to set this (and other services) to 1 replica?

Reply:
Tags are stateful. It's not HA regardless of the count. This was the value recommended by the maintaining squad.